### PR TITLE
jonathan fagan loops assignment

### PR DESCRIPTION
**Pull request by Jaiden16**

### CODE REVIEW

File: loops_lab.js [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+
+// 2. Write a `while` loop and a `for` loop that takes the variable "max", and iterates over all numbers from 0 to "max".
+// For each iteration, it will check if the current number is even or odd, and log that to the screen (e.g. "2 is even")
+//
+let max = 100;
+
+let num2 = 0;
+while (num2 <= max){
+num2 % 2 !== 0
```
Comment (jungraejang) [inline loops_lab.js]: Good use of ternary!

File: loops_lab.js [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+
+
+// 6. Write a loop that uses console.log to log all the numbers from 1 to 100, with two exceptions.
+//For numbers divisible by 3, log "Fizz" instead of the number, and for numbers divisible by 5 (and not 3), log "Buzz" instead.
+
+let num4 = 0;
+
+while (num4 <= 100){
+
+
```
Comment (jungraejang) [inline loops_lab.js]: Minor spacing issue. Make sure to make your code look pretty and readable for other users. Proper spacing and indentation do the job perfectly.

File: loops_lab.js [hunk 3]
```diff
@@ -0,0 +1,316 @@
+// # Loops Lab
```
Comment (jungraejang) [inline loops_lab.js]: Good demonstration of for and while loops overall. I like the use of ternary operators. Other than some spacing issues, everything looked fine. Good job!

**Comment by Jaiden16** [comment]
Thank you 😀

**Pull request closed without merging**